src/convertToObject.js

```diff
@@ -6,7 +6,22 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  if (sourceString.length === 0) {
+    return {};
+  }
+
+  const stringToObject = sourceString
+    .split(';')
+    .map((value) => {
+      return value.split(':').map((withoutSpace) => withoutSpace.trim());
+    })
+    .reduce((acc, value) => {
+      acc[value[0]] = value[1];
+
+      return acc;
+    }, {});
```

Comment:
The chained methods should have the same indentation to improve readability. Align the methods such as .split, .map, and .reduce to the same level.